docs/swagger.json

```diff
@@ -885,7 +885,7 @@
       }
     },
     "/profile/:userId/following": {
-      "post": {
+      "patch": {
```

Comment:
Why was this changed?

Reply:
I changed it because is a patch route and not a post route.

Reply:
What makes it a patch route?

Reply:
I don't really understand. How did we come to that conclusion?